Flow.Launcher/Languages/da.xaml

```diff
@@ -83,7 +86,7 @@
     <system:String x:Key="searchplugin_Noresult_Title">No results found</system:String>
     <system:String x:Key="searchplugin_Noresult_Subtitle">Please try a different search.</system:String>
     <system:String x:Key="plugin">Plugin</system:String>
-    <system:String x:Key="plugins">Plugins</system:String>
+    <system:String x:Key="plugins">Plugin</system:String>

```

Comment:
This needs to be corrected

Reply:
@taooceros should we just let this go and it will be corrected in the future via Crowdin with actual translation?